src/sandstorm/supervisor.capnp

```diff
@@ -274,15 +256,28 @@ struct ApiTokenOwner {
 
       introducerIdentity @9 :Text;
       # The identity ID through which a user's powerbox action caused the grain to receive this
-      # token. This is the identity against which the `requiredPermissions` parameter
-      # to `restore()` will be checked. This field is only intended to be filled in by the
-      # front-end during a powerbox request; a regular `save()` call produces a capability that
-      # has no "introducer".
+      # token. The token will become invalid if this identity loses access to the grain. This field
```

Comment:
I want an introducer identity to visualize which admin granted a grain an admin capability -- right now you can't get that from just a `userIsAdmin` membrane requirement.

I'd be okay with `userIsAdmin` turning from a `Text` into a struct or group with an `accountId` and an `identityId` and saying the requirements are the canonical location for this information.  But the issuing identity needs to be stored somewhere.